### DISCUSSION TITLE
Add AM | PM to Weekend Schedule & Where Applicable

### DISCUSSION THREAD

**Discussion by TheBaconCat** [f1multiviewer/issue-tracker#103] [Feature requests] [answered]
It would be useful to have AM | PM added to the Weekend Schedule and wherever else that would be applicable

![image](https://user-images.githubusercontent.com/11319293/178089589-b0a8a0a3-c762-4c06-9400-4788716bd6d8.png)

**Comment by AerodyneGhost** [comment]
Nah not really, 24 hour clock is the way for the future. :)

**Comment by jeppeolesen** [comment, marked as answer]
This has been implemented.
